### PR TITLE
Add rpc.discover method to Public API

### PR DESCRIPTION
**Pull request by aoudiamoncef**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [x] if the API has changed, update the API specification

### CODE REVIEW

**Comment by aoudiamoncef** [comment]
Related to https://github.com/massalabs/massa/issues/3184

**Comment by aoudiamoncef** [comment]
@AurelienFT is it useful to add this method to `massa-client` CLI ?

**Comment by AurelienFT** [comment]
> @AurelienFT is it useful to add this method to `massa-client` CLI ?

No.

File: massa-api/src/lib.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #![warn(unused_crate_dependencies)]
 use crate::error::ApiError::WrongAPI;
 use error::ApiError;
-use jsonrpc_core::{BoxFuture, IoHandler, Value};
+use jsonrpc_core::{BoxFuture, IoHandler, serde_json, Value};
```
Comment (AurelienFT) [inline massa-api/src/lib.rs:8]: I don't think usage related to this import

**Review by AurelienFT — COMMENTED** [review]
I think it also need a bit of formatting. Did you try the playground with your endpoint ? I think you can't because you don't have a domain name. Let's wait the end of the tests of consensus in labnet (ETA 2-3hours) and launch this in labnet to test it.

**Comment by aoudiamoncef** [comment]
You're right, I tested only via Postman. Sorry for formatting issues.

**Comment by AurelienFT** [comment]
Tested in labnet works well !

**Pull request merged**